Formula/node.rb

```diff
@@ -56,30 +56,11 @@ def install
 
     if build.with? "npm"
       resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
-      ENV.prepend_path "PATH", bin
```

Comment:
I'll go through this thread soon & reply to/pick out bits, but just wanted to note that the change on the `brew` side was literally thrown together in about 10 minutes last night to let me do some debugging. I'm considering a few options, that one was just easiest to test immediately 😄.